features/environment.py

```diff
@@ -223,7 +223,7 @@ def __init__(
         job_suffix = os.environ.get("UACLIENT_BEHAVE_JENKINS_BUILD_TAG")
         print("--- job suffix: {}".format(job_suffix), flush=True)
         if not job_suffix:
-            job_suffix = os.environ.get("TRAVIS_JOB_NUMBER", "dev")
+            job_suffix = os.environ.get("CHANGE_ID", "dev")
```

Comment:
@lucasmoura I'm not sure we get to this pathm because UACLIENT_BEHAVE_JENKINS_BUILD_TAG is set in Jenkinsfile already, so we don't fall into 'if not job_suffix' scope do we?